Update Dark Background

theck13 opened the pull request:
### Fix
Update the color values used in ***Dark*** theme to use darker shades of gray and follow [Material guidelines for elevation](https://material.io/design/color/dark-theme.html#properties).  See the screenshots below for illustration.

![update_dark_background](https://user-images.githubusercontent.com/3827611/65899070-2ea63e80-e370-11e9-96aa-3a33fda64d84.png)

### Test
0. Set theme to ***Dark***.
1. Open navigation drawer.
2. Notice navigation drawer background is lighter than navigation bar background.
3. Close navigation drawer.
4. Notice app bar background is lighter than note list background and status bar background.
5. Notice note list background is equal to navigation bar background.
6. Tap note in list with markdown enabled.
7. Tap ***Edit*** tab in app bar.
8. Notice app bar background is lighter than note content background and status bar background.
9. Tap ***Preview*** tab in app bar.
10. Notice app bar background is lighter than note content background and status bar background.
11. Notice note content background is equal in both ***Edit*** and ***Preview*** tabs.

### Review
Only one developer and one designer are required to review these changes, but anyone can perform the review.

### Release
`RELEASE-NOTES.txt` was updated in 83bbe974 with:
> Updated dark theme with darker colors for less eye fatigue and better battery life

SylvesterWilmott commented:
Looks fantastic @theck13 

There's a few spots where we can apply the new dark backgrounds as well:

- Note history bottom sheet
- Note info bottom sheet
- Note share bottom sheet

All of the above should be at 16dp elevation

We'll also want to make sure we use the lighter blue for the history sheet buttons for proper contrast.

theck13 commented:
I updated the bottom sheet background color in afa6e475.  I updated the flat button style in 57991208.  See the screenshots below for illustration.

![update_dark_background_sheet](https://user-images.githubusercontent.com/3827611/65970590-142d9d00-e424-11e9-9e98-ba92b391b0ab.png)

SylvesterWilmott commented:
Looks fantastic, thanks @theck13

AmandaRiu commented:
Passed design review. Merging.
